specs/lib/cacheOptimizeIndicesSpec.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var cacheOptimizeIndices = require('../../lib/cacheOptimizeIndices');
+var addDefaults = require('../../lib/addDefaults');
+var readGltf = require('../../lib/readGltf');
+var readAccessor = require('../../lib/readAccessor');
+
+var gltfPath = './specs/data/cacheOptimizeIndices/chinesedragon.gltf';
```

Comment:
How many triangles is this model?  To keep the repo small and the tests fast, why not just run this on a box?  The Cesium function is already unit tested, we just need to test the integration.